packages/astroport/src/lib.rs

```diff
@@ -25,4 +25,38 @@ mod uints {
     }
 }
 
+mod decimal_checked_ops {
```

Comment:
We need at least several test cases for these methods, as those are very important from math perspective and we need to be sure they work as expected

Reply:
Test added